core/vmpu/src/vmpu.c

```diff
@@ -27,6 +29,8 @@
 #include "vmpu_unpriv_access.h"
 #include <sys/reent.h>
 
+
+uint32_t g_debug_interrupt_sp[UVISOR_MAX_BOXES];
```

Comment:
We have another of `g_debug_interrupt_sp` in `core/debug/src/debug_box.c`. Why doesn't this cause multiple definition linker errors?